### PR TITLE
Refactor all tables to use MUI Table components

### PR DESCRIPTION
**Pull request by amiraliemami**
Reviewer: @lickem22 
Estimate: 30mins

---

## Ticket

Fixes: [808](https://idinsight.atlassian.net/browse/AAQ-808?atlOrigin=eyJpIjoiMmRjYTkxOGI5ODI2NDc0ZTgwZTYyYWFmOGNiOWE4MGUiLCJwIjoiaiJ9)

### Goal

- As title

## How has this been tested?

- Run dev or docker compose setup
- add some dummy data
- Go to dashboard page 1 and 2 and check all tables (including in popup sidebar on page 2)

## Checklist

Fill with `x` for completed. 

- [x] My code follows the style guidelines of this project
- [x] I have reviewed my own code to ensure good quality
- [x] I have tested the functionality of my code to ensure it works as intended
- [x] I have resolved merge conflicts

### CODE REVIEW

File: admin_app/src/app/dashboard/components/performance/ContentsTable.tsx [hunk 1]
```diff
@@ -1,15 +1,25 @@
...
 import Pagination from "@mui/material/Pagination";
 import TextField from "@mui/material/TextField";
-import Grid from "@mui/material/Unstable_Grid2";
 import { ApexOptions } from "apexcharts";
 import dynamic from "next/dynamic";
 import { useEffect, useState } from "react";
 
+import theme from "@/theme";
+import {
+  Paper,
```
Comment (lickem22) [inline admin_app/src/app/dashboard/components/performance/ContentsTable.tsx:13]: Unused imports

File: admin_app/src/app/dashboard/components/performance/ContentsTable.tsx [hunk 2]
```diff
@@ -180,113 +190,71 @@ const ContentsTable = ({
...
+          <TableRow>
+            <TableCell style={{ paddingTop: 0 }}>
+              <TextField
+                id="content-search"
+                placeholder="Search"
+                size="small"
+                sx={{
+                  width: "90%",
+                  marginTop: 1.5,
+                  bgcolor: "white",
```
Comment (lickem22) [inline admin_app/src/app/dashboard/components/performance/ContentsTable.tsx:212]: Should we delete if commented?

File: admin_app/src/app/dashboard/components/performance/ContentsTable.tsx (hunk 1 above)
Comment (lickem22) [reply admin_app/src/app/dashboard/components/performance/ContentsTable.tsx:13]: Nevermind

File: admin_app/src/app/dashboard/components/performance/ContentsTable.tsx (hunk 2 above)
Comment (lickem22) [reply admin_app/src/app/dashboard/components/performance/ContentsTable.tsx:212]: Nevermind again. I had outdated code.

File: admin_app/src/app/dashboard/components/performance/DetailsDrawer.tsx [hunk 3]
```diff
@@ -203,83 +213,45 @@ const DetailsDrawer: React.FC<DetailsDrawerProps> = ({
...
-                whiteSpace: "nowrap",
-                textOverflow: "ellipsis",
-                lineHeight: "20px",
-              }}
-            >
-              {feedback.feedback}
-            </Grid>
-          </Grid>
-        ))}
+        <TableContainer component={Paper} sx={{ mt: 2 }}>
```
Comment (lickem22) [inline admin_app/src/app/dashboard/components/performance/DetailsDrawer.tsx:216]: Just curious, what are the advantages of `TableContainer` and  `Table` compared to `Grid`?

Comment (amiraliemami) [reply admin_app/src/app/dashboard/components/performance/DetailsDrawer.tsx:216]: We had a bug where cells of each row would resize weirdly smaller than the "s" screen size, also this is the exact intended usecase of this component so makes sense to use it directly!

**Review by lickem22 — APPROVED** [review]
Thanks @amiraliemami, Works as expected. I am approving. Feel free to merge whenever

**Pull request merged**